src/stores/AppStore.js

```diff
@@ -232,9 +232,11 @@ export default class AppStore extends Store {
       debug('System resumed, last suspended on', this.timeSuspensionStart);
       this.actions.service.resetLastPollTimer();
 
+      const idleTime = this.stores.settings.app.reloadAfterResumeTime;
+
       if (
-        this.timeSuspensionStart.add(10, 'm').isBefore(moment()) &&
-        this.stores.settings.app.get('reloadAfterResume')
+        this.timeSuspensionStart.add(idleTime, 'm').isBefore(moment()) &&
+        this.stores.settings.app.reloadAfterResume
```

Comment:
This is an integer representing time in minutes - if so, then this if condition logic is incorrect